### ISSUE TITLE
Suggestion: Simplify rune creation/execution calls

### ISSUE THREAD

**Issue by d4amenace** [ElementsProject/lightning#6047]
Hello,

I have noticed that the cmds for rune creation "commando rune" and "commando" have very complext JSON format req's.  In a future update of the protocol, can we get a simplified format for  creating these calls?  One such example of a rune call with args is:

`lightning-cli commando peer_id=05161691907069013aa8a43feed9417dc5c85a5939f7036e3490ac3aab3fe43d9 method=invoice params="{ \"amount_msat\": 1000, \"label\": \"ajdkjalj\", \"description\": \"testing\" }" rune=zThVwe0rMNI1DNaVzrqSwOV_hRHiohUuiijiOoSZwbnVtPDY= `

As you can see, in JSON format, the call is highly dependent on proper JSON format for extended args.  The average user will have issues setting up complex rune-calls with JSON formatted extended args.

This is just the issue with execution.  The JSON formatting for restrictions is quite complex and I have found that it is easier to first create an un-restricted rune, then add restrictions one-by-one, until the final rune has all restrictions.  This can be done with a single call upon rune creation, but the complexity of formatting to JSON for multiple restriction args is complex.

**Comment by vincenzopalazzo** [comment]
What format do you suggest instead of common json??

**Comment by rustyrussell** [comment]
Yes, I think rune authoring is best done with a tool.  I'm probably not the best person to write such a tool though!

**Comment by d4amenace** [comment]
> What format do you suggest instead of common json??

Good point.

**Comment by sangbida** [comment]
This issue is being closed as part of issue cleanup.

**Reason**: No recent activity (last updated 1018 da

If you believe this issue should remain open, please comment and we can reopen it.
